readthedocs.yml

```diff
@@ -0,0 +1,7 @@
+python:
+    version: 3.5
+    setup_py_install: true
+    pip_install: true
+    extra_requirements:
+        - tests
+        - docs
```

Comment:
docs requirements are not read. Should add sth in setup.py https://github.com/pysal/pysal/blob/master/setup.py#L48